Python/specialize.c

```diff
@@ -2393,12 +2394,12 @@ _Py_Specialize_CompareOp(_PyStackRef lhs_st, _PyStackRef rhs_st, _Py_CODEUNIT *i
         goto failure;
     }
     if (PyFloat_CheckExact(lhs)) {
-        instr->op.code = COMPARE_OP_FLOAT;
+        specialized_op = COMPARE_OP_FLOAT;
```

Comment:
Now you can use helpers to write it, like `specialize(instr, COMPARE_OR_FLOAT)` which looks clearer.
(You need to merge main to get changes from 9ce4fa0719d291070b6a66fe25716ef1e81448f)

Reply:
Done.